src/engraving/libmscore/noteentry.cpp

```diff
@@ -203,7 +203,7 @@ Note* Score::addPitch(NoteVal& nval, bool addFlag, InputState* externalInputStat
         //   We could split the duration at the barline and continue into the next bar, but this would create extra
         //   notes, extra ties, and extra pain. Instead, we simply truncate the duration at the barline.
         Fraction ticks2measureEnd = is.segment()->measure()->ticks() - is.segment()->rtick();
-        duration = is.duration() > ticks2measureEnd ? ticks2measureEnd : is.duration().fraction();
+        duration = is.duration().fraction() > ticks2measureEnd ? ticks2measureEnd : is.duration().fraction();
```

Comment:
This is where I was saying I would imagine that `.ticks()` instead of `.fraction() `  would be more appropriate here, but I've not found a problem with Note Entry through a MIDI device in master branch as with #9000...

Reply:
As per https://github.com/Jojo-Schmitz/MuseScore/commit/3681b72032f35e731ff3e2bc933528a9ae594777#commitcomment-75143487 this has been a false alert 😉